tests/IntegrationTests/TokenAcquirerTests/TokenAcquirer.cs

```diff
@@ -126,6 +127,46 @@ public async Task AcquireToken_WithFactoryAndAuthorityClientIdCert_ClientCredent
             Assert.False(string.IsNullOrEmpty(result.AccessToken));
         }
 
+        [IgnoreOnAzureDevopsFact]
+        //[Fact]
```

Comment:
Should this comment be kept if the other attribute is being used?

Reply:
Kept the pattern followed by other tests in the file.